### DISCUSSION TITLE
part-ii/2014-124/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#38] [Question comments]
Part II 2014 Probability and Measure: Paper 3, Section II, K 

https://questions.tripos.org/part-ii/2014-124/

**Comment by ArthurConmy** [comment]
$\theta$ is measure preserving since we can verify this on the $\pi$ system of intervals $[a,b] \times [c,d]$ which generate the sigma algebra and conclude by Dynkin.

$\theta$ is not ergodic, since the set $\{ (x_1, x_2) : x_1 - x_2 \in (0, \frac12 ) \pmod{1} \}$ is $\theta$-invariant and doesn't have measure zero.

I believe

$$ \bar{f}(x_1,x_2) = \int_{\{ (\tilde{x_1}, \tilde{x_2}) : x_1 - x_2 = \tilde{x}_1 - \tilde{x}_2 \pmod{1} \}} f $$

works, but I'm a bit shaky on justifying that we can do "1D integration" with the 2D function $f$.
